crates/interledger-stream/src/server.rs

```diff
@@ -84,25 +85,34 @@ impl ConnectionGenerator {
     }
 }
 
+/// A payment notification data type to be used by Pubsub API consumers
 #[derive(Debug, Deserialize, Serialize)]
 pub struct PaymentNotification {
+    /// The username of the receiver of the payment notification
     pub to_username: Username,
+    /// The username of the sender of the payment notification
```

Comment:
Is this more accurate?

```suggestion
    /// The username of the account that routed the Interledger payment to this node
```